README.md

```diff
@@ -22,6 +22,8 @@
 - Please read _all_ the links included:
 - Take a look at [our **Guidebook**](https://bwtc-handbook.notion.site/BWTC-Handbook-71256770d53f46439aecd989b07d1533) for all your (mostly) non-code related needs!
 
+Get set up with [Github](/standards/git-config.md)
```

Comment:
This is such a nitpick....but I think this should be **GitHub**
```suggestion
Get set up with [GitHub](/standards/git-config.md)
```

I don't think of this as a blocker, but it would be nice to stay consistent with company names. We probably have `Github` all over the S&P to be honest.